### PR TITLE
[JAVA] Codestyle check

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

File: inference-engine/ie_bridges/java/scripts/codestyle_check.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cd ~/openvino/bin/intel64/Release/lib/
+wget -nc https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+
+java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed |
+     -r ~/openvino/inference-engine/ie_bridges/java/tests/BlobTests.java ~/openvino/inference-engine/ie_bridges/java/tests/BlobTests.java
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/scripts/codestyle_check.sh:5]: Please move all the code to `.yml` file

File: .github/workflows/code_style_java.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Code Style Java
+on: [push, pull_request]
+
+jobs:
+  InferenceEngine:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
```
Comment (dkurt) [inline .github/workflows/code_style_java.yml:10]: We do not need sumbodules

File: .github/workflows/code_style_java.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Code Style Java
```
Comment (dkurt) [inline .github/workflows/code_style_java.yml:1]: Please create a separate job instead in [code_style.yml](https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/code_style.yml):
```yml
jobs:
  nGraph:
  ....
  Java:
  <your code>
```

File: .github/workflows/code_style_java.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Check java code style
+        run: |
+          inference-engine/ie_bridges/java/scripts/codestyle_check.sh
+          git diff >java_code_style_diff.patch
```
Comment (dkurt) [inline .github/workflows/code_style_java.yml:15]: `git diff` is a step which should be performed in case of failure of code style check

**Comment by dkurt** [comment]
If the workflow did not start - go to https://github.com/openvinotoolkit/openvino/actions and fix syntax errors. In example, https://github.com/openvinotoolkit/openvino/actions/runs/225956273

**Comment by azhogov** [comment]
This code style check is running on push into forks too, so, you can develop it without creating a PR.
Also I see no "check". What is the criteria that style should be fixed?

**Pull request closed without merging**